samples/Basic_NFC/app/MFRC522.cpp

```diff
@@ -0,0 +1,1768 @@
+/*
```

Comment:
A better location for MFRC522.cpp and MFRC522.h will be under `Sming/Libraries/MFRC522`. This way also other applications can use those libraries just by including Sming.